SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -236,6 +236,8 @@ namespace SpatialConstants
 	const FString SCHEMA_DATABASE_FILE_PATH  = TEXT("Spatial/SchemaDatabase");
 	const FString SCHEMA_DATABASE_ASSET_PATH = TEXT("/Game/Spatial/SchemaDatabase");
 
+	static const FString ZoningAttribute = DefaultServerWorkerType.ToString();
```

Comment:
Nothing in this file needs to be static. Can you do a quick pass?

Reply:
Done